Makefile

```diff
@@ -33,15 +42,14 @@ MAIN_TEST_FILES := pkg/dcgmexporter/system_info_test.go
 all: ubuntu20.04 ubi8
 
 binary:
-	cd cmd/dcgm-exporter; go build -ldflags "-X main.BuildVersion=${DCGM_VERSION}-${VERSION}"
+	cd cmd/dcgm-exporter; $(GO) build $(GOFLAGS) -ldflags "-X main.BuildVersion=${DCGM_VERSION}-${VERSION}"
 
 test-main: $(NON_TEST_FILES) $(MAIN_TEST_FILES)
-	go test ./...
+	$(GO) test ./...
 
-install: binary
-	install -m 557 cmd/dcgm-exporter/dcgm-exporter /usr/bin/dcgm-exporter
-	install -m 557 -D ./etc/default-counters.csv /etc/dcgm-exporter/default-counters.csv
-	install -m 557 -D ./etc/dcp-metrics-included.csv /etc/dcgm-exporter/dcp-metrics-included.csv
+install:
+	install -m 0755 -D -t $(BINDIR) cmd/dcgm-exporter/dcgm-exporter
+	install -m 0644 -D -t $(SYSCONFDIR) etc/default-counters.csv etc/dcp-metrics-included.csv
```

Comment:
dcgm-exporter by default will look for the default-counters.csv in the /etc/dcgm-exporter/ location:
https://github.com/NVIDIA/dcgm-exporter/blob/0869a2ac03c73e82715438f44d2ecf2491049139/cmd/dcgm-exporter/main.go#L96

Reply:
You're right - that's an error on my part. Will fix.